### ISSUE TITLE
Config file does not load before `tryGetFileContents`

### ISSUE THREAD

**Issue by yunsii** [iiian/prisma-case#62]
https://github.com/iiian/prisma-case-format/blob/fbdb03fd56f01ee5195ce9b1cfd7f7b7a616b5f0/src/cli.ts#L36-L43

If I fully depend on default config file with custom `file`, it doesn't work.

**Comment by iiian** [comment]
Hey @yunsii. I want to apologize to you, I haven't had time to get around to addressing the issues you raised 😞. Unfortunately, I don't have a ton of time to maintain this project right now, but hopefully would around May or June. I want to try to help you address your issue: have you tried using an older version with a thinner feature set? I know that's not ideal but maybe it could unblock you 😕 sorry I don't have better news.

**Comment by yunsii** [comment]
Previous PR https://github.com/iiian/prisma-case-format/pull/61 sill not merged, Maybe I can create a new PR if that PR merged 😂
